### ISSUE TITLE
[cpu] plugin not work with variables?

### ISSUE THREAD

**Issue by bcg62** [redhat-performance/tuned#203]
```
$ cat /tmp/test4
1
```

```
[variables]
no_turbo=${f:strip:${f:exec:cat:/tmp/test4}}
```

```
[cpu]
no_turbo=${no_turbo}
```

`2019-10-29 17:02:14,945 ERROR    tuned.utils.commands: Writing to file '/sys/devices/system/cpu/intel_pstate/no_turbo' error: '[Errno 22] Invalid argument'`

I've enabled debug and see it as what i would expect.
`('TUNED_no_turbo', '1')`

tuned-2.10.0-15.el8.noarch

**Comment by bcg62** [comment]
using the following settings without the variable works as expected.

```
[cpu]
no_turbo=1
```

```
$ cat /sys/devices/system/cpu/intel_pstate/no_turbo
1
```

**Comment by olysonek** [comment]
Thanks for the report. Should be fixed in #204.
